source/system/Demo_User_Guides/TI_Apps_Launcher_User_Guide.rst

```diff
@@ -365,14 +365,70 @@ _________________________
 
 3. Once you enter the necessary values for both HTTPS and No proxy, click on `Set Proxy` to set the proxy configuration.
 
+
+Using terminal
+______________
+
+.. ifconfig:: CONFIG_sdk in ('SITARA')
+
+   The terminal button on the left panel can be used to open a terminal application, which is
+   based on QMLtermwidget.
+
+   .. Image:: /images/terminal.png
+      :height: 400
+
+.. ifconfig:: CONFIG_sdk not in ('SITARA')
```

Comment:
Why not put this whole section inside the ifconfig rather than suggest that it is not available?

Reply:
@AashvijShenai Not sure but i am observed that sphinx does not support declaring a new subsection inside a ifconfig. I added a test section
```
.. ifconfig:: CONFIG_part_variant in ('AM62X', 'AM62PX')                                                                                                  

    Test Menu                                                                    
     ^^^^^^^^^                                                                                                                                                     
    This is a test section
```
The test Menu appeared as a new section after build. (And not a sub-section which is required). If you can try it on your end as well.